src/handlers/gestureHandlerCommon.ts

```diff
@@ -180,3 +181,17 @@ export function findNodeHandle(
   if (Platform.OS === 'web') return node;
   return findNodeHandleRN(node);
 }
+
+let scheduledHandlersQueueFlush:
+  | ReturnType<typeof requestAnimationFrame>
+  | undefined = undefined;
```

Comment:
Can we use `null` as the default value instead? Then we can use `?` from TypeScript instead of `| undefined`.

Reply:
TS didn't really like the idea of using `?` here, but using `null` instead of `undefined` is better. Done in https://github.com/software-mansion/react-native-gesture-handler/pull/1979/commits/325994e3843a700f3f01ae6f1c5c251f2c692c64.